### DISCUSSION TITLE
Full text search when using tags

### DISCUSSION THREAD

**Discussion by ankyfive** [requarks/wiki#3473] [Help / Questions]
**Question**
Hello,
Sorry for all my questions !!

We have activated advanced postgres search which gives the possibility to have full text search.

But when we use tags it does not work... Is there a way to also use full text search with tags ?
An example with your doc :
Go there : https://docs.requarks.io/t/docker?sort=title
and search Compose or hostname...

Thanks :-)

**Host Info (please complete the following information):**
OS: Debian 9.12
Wiki.js version: 2.5.159
Database engine:postgres 9.6.17

**Comment by despens** [comment]
See #2138 and #1729

**Comment by R0Wi** [comment]
I'm on Wiki.js 2.5.170 with Elasticsearch as search engine and i'm facing the same problem. I had a look at the Elastic index and for me it seems if the tags are just not indexed. Could that be a missing feature maybe? Would love to have this implemented, let me know if i can support you somehow.
